### PR TITLE
{vis}[GCCcore/12.3.0] Gdk-Pixbuf v2.42.10

### PR DESCRIPTION
**Pull request by stefan-wolfsheimer**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
@stefan-wolfsheimer easyconfig for `LibTIFF/4.5.1-GCCcore-12.3.0` is missing here?

**Comment by stefan-wolfsheimer** [comment]
> @stefan-wolfsheimer easyconfig for `LibTIFF/4.5.1-GCCcore-12.3.0` is missing here?

@boegel, the library was installed in our cluster without PR. Sorry for that.
The dependency should be resolved with this PR:

https://github.com/easybuilders/easybuild-easyconfigs/pull/18627

File: easybuild/easyconfigs/g/Gdk-Pixbuf/Gdk-Pixbuf-2.42.10-GCCcore-12.3.0.eb [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    ('Ninja', '1.11.1'),
+    ('pkgconf', '1.9.5'),
+    ('GObject-Introspection', '1.76.1'),
+]
+
+dependencies = [
+    ('GLib', '2.77.1'),
+    ('libjpeg-turbo', '2.1.5.1'),
+    ('libpng', '1.6.39'),
+    ('LibTIFF', '4.5.1'),
```
Comment (SebastianAchilles) [inline easybuild/easyconfigs/g/Gdk-Pixbuf/Gdk-Pixbuf-2.42.10-GCCcore-12.3.0.eb:32]: We already have an EasyConfig for LibTIFF v4.5.0: https://github.com/easybuilders/easybuild-easyconfigs/blob/develop/easybuild/easyconfigs/l/LibTIFF/LibTIFF-4.5.0-GCCcore-12.3.0.eb

Usually we try to use one version per toolchain generation, unless there is the requirement for a newer versions.

```suggestion
    ('LibTIFF', '4.5.0'),
```

Comment (boegel) [reply easybuild/easyconfigs/g/Gdk-Pixbuf/Gdk-Pixbuf-2.42.10-GCCcore-12.3.0.eb:32]: Oh, I overlooked that...

Indeed, we're already using LibTIFF 4.5.0 in several easyconfigs using `GCCcore/12.3.0`, so we should stick to that...

Comment (stefan-wolfsheimer) [reply easybuild/easyconfigs/g/Gdk-Pixbuf/Gdk-Pixbuf-2.42.10-GCCcore-12.3.0.eb]: I changed the dependency for this package accordingly and will retract my PR for libTIFF 4.5.1
https://github.com/easybuilders/easybuild-easyconfigs/pull/18627

**Comment by boegel** [comment]
@boegelbot please test @ generoso

**Review by boegel — APPROVED** [review]
lgtm

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in total)
node3118.skitty.os - Linux RHEL 8.6, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/boegel/6f8d70191dea4e308ecc1e2679ffc31c for a full test report.

**Comment by boegel** [comment]
@boegelbot please test @ jsc-zen2

**Comment by boegel** [comment]
Going in, thanks @stefan-wolfsheimer!

**Pull request merged**